crisischeckin/crisicheckinweb/Views/Shared/_Layout.cshtml

```diff
@@ -45,7 +45,7 @@
         @RenderBody()
     </div>
     
-    <script src="http://ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.1.min.js"></script>
+    <script src="//ajax.aspnetcdn.com/ajax/jquery/jquery-2.1.1.min.js"></script>

```

Comment:
Am I missing something? It looks like this url won't work. Doesn't it need the http: as well?